orchestrator/src/main/kotlin/WorkerScheduleContext.kt

```diff
@@ -53,7 +53,14 @@ internal class WorkerScheduleContext(
     /**
      * A map storing the jobs for the current run that have already been created. Keys are the endpoint names for jobs.
      */
-    val jobs: Map<String, WorkerJob>
+    val jobs: Map<String, WorkerJob>,
+
+    /**
+     * A flag indicating that the current [OrtRun] has failed. Per default, the failure state is determined by the
+     * jobs that have been run. With this flag, this mechanism can be overridden, which is necessary for workers that
+     * do not spawn jobs.
```

Comment:
I don't understand this explanation, under what circumstance would this be set to `true`?

Reply:
The comment has been extended to mention the Config worker which is currently the only example.